### ISSUE TITLE
Python 3.9 Support

### ISSUE THREAD

**Issue by ndgayan** [rapidsai/dask-cuda#595]
Could you add this?

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - dask-cuda -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']

Your python: python=3.9

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.

**Comment by jakirkham** [comment]
Yeah RAPIDS doesn't currently support Python 3.9. It is planned, but I don't believe it has been scheduled yet.

**Comment by pentschev** [comment]
RAPIDS now supports Python 3.9, closing.
